### PR TITLE
Update meeting details contract

### PR DESCRIPTION
**Pull request by ronakbakshi**
There is a request to update the API contract

### CODE REVIEW

File: src/public/meeting.ts [hunk 1]
```diff
@@ -8,6 +8,24 @@ export namespace meeting {
    * Data structure to represent a meeting details.
    */
   export interface IMeetingDetails {
+    /**
+     * details object
+     */
+    details: IDetails;
```
Comment (ydogandjiev) [inline src/public/meeting.ts:14]: Seems a little redundant to have an `IDetails` object within an `IMeetingDetails` object. Should we maybe rename the outer interface to `IMeetingProperties` and change the SDK API name to `getMeetingProperties`? What is the reason for creating this additional nesting?

**Review by ydogandjiev — COMMENTED** [review]
@ronakbakshi, can you please add some more details to the description of this PR? Are we changing this interface to more closely match the response in the Bot SDK? If so, could you please share some information on the Bot SDK API and its interface that we are mimicking here?

File: src/public/meeting.ts (hunk 1 above)
Comment (ronakbakshi) [reply src/public/meeting.ts:14]: @ydogandjiev we are working on making our response like this 
`{​​​​
  "details": {​​​​
     "id": " MCMxOTptZWV0aW5nX05EUXhNemcxWWpVdE1HSXpOQzAwWWpjMUxXRm1ZV1l0WXprMU1HWTJNVE13TmpFMEB0aHJlYWQudjIjMA==",
    "scheduledStartTime": "2020-08-21T02:30:00+00:00",
    "scheduledEndTime": "2020-08-21T03:00:00+00:00",
    "JoinUrl": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_NDQxMzg1YjUtMGIzNC00Yjc1LWFmYWYtYzk1MGY2MTMwNjE0%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%226aebbad0-e5a5-424a-834a-20fb051f3c1a%22%7d",
    "title": "Teams All Hands",
    "type": "Scheduled"
  }​​​​,
  "conversation": {​​​​
    "id": "19:meeting_NDQxMzg1YjUtMGIzNC00Yjc1LWFmYWYtYzk1MGY2MTMwNjE0@thread.v2"
  }​​​​,
  "organizer": {​​​​
    "id": "29:1JKiJGPAX9TTxtGxhVo0wLx_zwzo-gG8Z-X03306vBwi9p-xMTEbDXsT6KH7-0kkTS8cD-2zkrsoV6f5WJ6_aYw",
    "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47"
  }​​​​
}​​​​`

Comment (ronakbakshi) [reply src/public/meeting.ts:14]: As discussed in the chat thread, I am marking these comments as resolved. @ydogandjiev please let me know if you have any other feedback :)

**Review by ydogandjiev — APPROVED** [review]

File: test/public/meeting.spec.ts [hunk 2]
```diff
@@ -180,14 +180,26 @@ describe('meeting', () => {
...
+        type: meeting.MeetingType.Scheduled,
+      };
+      let organizer: meeting.IOrganizer = {
+        id: '8:orgid:6b33ac33-85ae-4995-be29-1d38a77aa8e3',
+        tenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47',
+      };
+      const conversation: meeting.IConversation = {
+        id: `convId`,
+      };
+      let meetingDetails: meeting.IMeetingDetails = {
```
Comment (ydogandjiev) [inline test/public/meeting.spec.ts:199]: NIT: `let` --> `const`

File: test/public/meeting.spec.ts [hunk 3]
```diff
@@ -180,14 +180,26 @@ describe('meeting', () => {
...
         scheduledEndTime: '2020-12-21T22:00:00+00:00',
-        meetingTitle: 'Get meeting details test meeting',
-        organizerId: '8:orgid:6b33ac33-85ae-4995-be29-1d38a77aa8e3',
-        tenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47',
         joinUrl:
           'https://teams.microsoft.com/l/meetup-join/19%3ameeting_qwertyuiop[phgfdsasdfghjkjbvcxcvbnmyt1234567890!@#$%^&*(%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%226b33ac33-85ae-4995-be29-1d38a77aa8e3%22%7d',
+        title: 'Get meeting details test meeting',
+        type: meeting.MeetingType.Scheduled,
+      };
+      let organizer: meeting.IOrganizer = {
```
Comment (ydogandjiev) [inline test/public/meeting.spec.ts:192]: NIT: `let` --> `const`

File: test/public/meeting.spec.ts [hunk 4]
```diff
@@ -180,14 +180,26 @@ describe('meeting', () => {
       let getMeetingDetailsMessage = desktopPlatformMock.findMessageByFunc('meeting.getMeetingDetails');
       expect(getMeetingDetailsMessage).not.toBeNull();
       let callbackId = getMeetingDetailsMessage.id;
-      let meetingDetails: meeting.IMeetingDetails = {
+      let details: meeting.IDetails = {
```
Comment (ydogandjiev) [inline test/public/meeting.spec.ts:183]: NIT: `let` --> `const`

**Review by ydogandjiev — APPROVED** [review]
Left some minor comments.

File: test/public/meeting.spec.ts (hunk 2 above)
Comment (ronakbakshi) [reply test/public/meeting.spec.ts]: Updated. Thanks

File: test/public/meeting.spec.ts (hunk 4 above)
Comment (ronakbakshi) [reply test/public/meeting.spec.ts]: Updated. Thanks

**Pull request merged**